tools/sgcollect_info

```diff
@@ -607,8 +564,12 @@ def make_sg_tasks(zip_dir, sg_url, sync_gateway_config_path_option, sync_gateway
     # Add a task to collect Sync Gateway config
     config_tasks = make_config_tasks(zip_dir, sg_config_path, sg_url, should_redact)
 
-    # Curl the / endpoint and /db endpoints and save output
-    status_tasks = make_status_tasks(sg_url, should_redact)
+    # Curl the /_status
+    status_tasks = make_curl_task(name="Collect server status",
+                                  user="",
+                                  password="",
+                                  url="{0}/_status".format(sg_url),
+                                  log_file="sync_gateway.log")
```

Comment:
If multiple tasks are assigned to the same log_file, will they be appended, or overwritten? (looks like Running server config above got renamed to match this one)

Reply:
They get appended to the file.